CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.17.3-dev
 * Bug - Fixed a bug where a stale websocket connection could linger [#1310](https://github.com/aws/amazon-ecs-agent/pull/1310)
+* Enhancement - Introduce InactivityTimeout to image pulling [#1290](https://github.com/aws/amazon-ecs-agent/pull/1290)
```

Comment:
This should be more colloquial in nature. Example:
`Enhancement - Fail image pulls if there's inactivity during image pull progress ...`. Also, make sure that `Enhancement`s precedes `Bug`s